test/problems/identity/generators/generators.yaml

```diff
@@ -169,16 +169,8 @@ data:
           - b: stdout.py 502 # 2-b
       non_empty_group_6:
         data:
-          - "": stdout.py 503
-      json:
-        {
-          data:
-            [
-              { "": stdout.py 600 },
-              { "1": stdout.py 601 },
-              { "": stdout.py 602 },
-            ],
-        }
+          - '': stdout.py 503
+      json: {data: ['': stdout.py 600, '1': stdout.py 601, '': stdout.py 602]}
```

Comment:
Can you revert this change? I think the extra newlines and indentation are helpful to understanding the syntax :slightly_smiling_face: